### ISSUE TITLE
accuracy metric while training

### ISSUE THREAD

**Issue by shubham-IISc** [espnet/espnet#224]
Really incredible work. While training, in log file, I could not figure out what exactly is accuracy metric being displayed. Can it be changed to WER/CER while training as I would like to observe how my WER/CER vary with number of epochs on validation set?

**Comment by sw005320** [comment]
I tried to make the main terminal as simple as possible.
One way to monitor the accuracy, loss, etc. during training is to use the following:
```
tail -f exp/${expdir}/train.log
```
I'll add it to https://github.com/espnet/espnet/blob/master/README.md

**Comment by kan-bayashi** [comment]
Hi @shubham-IISc,
Maybe you want to check CER or WER for each epoch instead of accuracy, right?
Currently, we use sclite to calculate CER and WER, therefore it is a little bit difficult to integrate it with training process.
If there is nice python library which easily calculates CER, maybe we can use it.
Do you know such library?

**Comment by pozhijisi** [comment]
xer [(https://github.com/jpuigcerver/xer)](https://github.com/jpuigcerver/xer) maybe the one you are looking for. @kan-bayashi

**Comment by sw005320** [comment]
Since we use teacher forcing during the training, the WER/CER is not correct anyway, and I feel like accuracies are good enough. But, actually @mn5k is working on incorporating the computation of the WER/CER during the training for his MWER training implementation (he performs decoding and exact WER/CER computation during training). So, once his implementation is fixed, we can use his development to compute them (he seems to use some existing edit distance computation tools).

**Comment by shubham-IISc** [comment]
Hi @kan-bayashi  actually when I implemented CTC in tensorflow, I implemented my own CER python script based on minimum edit distance. But I believe, xer suggested by @pozhijisi could be helpful as well.

**Comment by shubham-IISc** [comment]
@sw005320 yes i agree that WER/CER is not correct to evaluate on training set, but after each epoch, we may like to recognize and decode development set and observing WER/CER on it may give us better analysis of model fitting as our ultimate metric is WER once, training is completed. Correct me, if I am  wrong. thanks.

**Comment by sw005320** [comment]
I see the point. @mn5k, do you think it is possible to add your WER/CER computation during training (for training and validation data)? This would provide more detailed information when monitoring training in addition to accuracies and losses.

**Comment by mn5k** [comment]
Yes, it is possible. I'll implement the option to monitor WER/CER.
